GrzybekApk/app/src/main/java/com/example/grzybekapk/view/fragments/FragStartScreen.kt

```diff
@@ -31,21 +37,51 @@ class FragStartScreen : Fragment(){
         val eventNameArray = ArrayList<String>() // Array for event names from database
         val organizerNameArray = ArrayList<String>() // Array for event organizers from database
         val eventDescriptionArray = ArrayList<String>() // Array for event description from database
-        //val eventDateArray = ArrayList<Date>()  Array for event date from database ???
-        val arrayAdapter = ArrayAdapter(activity,android.R.layout.simple_list_item_1,eventNameArray)
+        val eventDateArray = ArrayList<String>()   // Array for event date from database
+        val eventsArray = ArrayList<String>()   // Array for event name and date for list view
+        val arrayAdapter = ArrayAdapter(activity,android.R.layout.simple_list_item_1,eventsArray)
         eventsListView.adapter = arrayAdapter
 
-        eventNameArray.add("Flanki")                    // hardcoded for testing, waiting for backend boys ( ͡° ͜ʖ ͡°)
-        organizerNameArray.add("Mati")
-        eventDescriptionArray.add("Final")
-        eventNameArray.add("Grill")                    // hardcoded for testing, waiting for backend boys ( ͡° ͜ʖ ͡°)
-        organizerNameArray.add("Barti")
-        eventDescriptionArray.add("Grillowanie na grzybku")
+        //tak to musi być tak pojebane inaczej nie działa
```

Comment:
Xd

Reply:
nawet nie wiesz ile się z tym męczyłem żeby zrobić to na jednej klasie ale nigdy nie chciało działać